common/src/main/java/org/apache/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -43,13 +43,25 @@
  *
  */
 public class EnhancedServiceLoader {
+    private static final String APACHE_SEATA_PACKAGE_NAME = "org.apache.seata";
+    private static final String IO_SEATA_PACKAGE_NAME = "io.seata";
 
     /**
      * Class->InnerEnhancedServiceLoader map
      */
     private static final ConcurrentMap<Class<?>, InnerEnhancedServiceLoader<?>> SERVICE_LOADERS =
             new ConcurrentHashMap<>();
 
+    private static <S> Class<S> getCompatibleService(Class<S> originService) {
+        String apacheSeataName = originService.getName();
+        String ioSeataName = apacheSeataName.replace(APACHE_SEATA_PACKAGE_NAME, IO_SEATA_PACKAGE_NAME);
```

Comment:
Explicitly identify the Apache Seata SPI using sartwith or other methods.

Reply:
If both exist, what is the priority?

Reply:
If both package names exist, the current one is based on io. seata, and the package for org. apache. seata will only be loaded if it cannot be loaded. In addition, I believe that there should not be two types of package names at the same time, as compatibility itself is to provide users with a transitional solution, not a persistent compatibility solution. If users need to customize a new SPI implementation, they should switch to the package name of Apache seata as soon as possible.